app/views/products/show.html.erb

```diff
@@ -42,21 +36,37 @@
 						<div class="col-sm-7">
 							<div class="product-information"><!--/product-information-->
 								<img src="images/product-details/new.jpg" class="newarrival" alt="">
+
 								<h2><%= @product.name %></h2>
 								<p>Product code: <%= @product.id %></p>
 								<img src="images/product-details/rating.png" alt="">
-								<span>
-									<span>US $59</span>
-									<label>Quantity:</label>
-									<input type="text" value="3">
-									<button type="button" class="btn btn-fefault cart">
-										<i class="fa fa-shopping-cart"></i>
+								<span >
+									<span ><%=  @product.product_details.pluck(:price)[0] %>$ </span>
+									
+									<label><%= t".Quantity" %>:</label>
+									
+										<button type="button" class="btn btn-fefault cart">
+											<i class="fa fa-shopping-cart"></i>
 										Add to cart
 									</button>
+
+									<input type="text" value=""><br>
+									<h3>color:</h3>
+									<% option_for_color.map do |c| %>
```

Comment:
dùng hẳn option for select và hiển thị n như là 1 button nhé anh
tham khảo ở đây nhé
https://stackoverflow.com/questions/52707749/how-can-i-display-select-options-as-buttons